component/grant-type/pom.xml

```diff
@@ -0,0 +1,144 @@
+                            org.wso2.carbon.identity.oauth2.*;
+                            version="${carbon.identity.oauth.package.import.version.range}",
+                            org.wso2.carbon.identity.oauth.*;
+                            version="${carbon.identity.oauth.package.import.version.range}",
+                            org.wso2.carbon.identity.base; version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.core.util;
+                            version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.application.authentication.framework.*,
+                            org.wso2.carbon.utils.*; version="${carbon.kernel.package.import.version.range}",
+                            com.nimbusds.jose.*; version="${nimbusds.osgi.version.range}",
+                            com.nimbusds.jwt; version="${nimbusds.osgi.version.range}",
+                            org.wso2.carbon.identity.application.common.*;
+                            version="${carbon.identity.package.import.version.range}",
+                            org.apache.commons.lang; version="${commons-lang.wso2.osgi.version.range}",
+                            org.apache.commons.logging; version="${commons-logging.osgi.version.range}",
+                            org.wso2.carbon.idp.mgt.*; version="${carbon.identity.framework.version}",
+                            org.wso2.carbon.user.core.service;version="${carbon.kernel.package.import.version.range}",
+                            net.minidev.json; version="${net.minidev.json.imp.pkg.version.range}",
+                            org.wso2.carbon.context; version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.user.core.util; version="${carbon.kernel.package.import.version.range}",
+                            org.apache.commons.collections; version="${commons-collections.wso2.osgi.version.range}"
+                        </Import-Package>
+                        <Export-Package>
+                            !org.wso2.carbon.identity.oauth2.grant.token.exchange.internal,
+                            org.wso2.carbon.identity.oauth2.grant.token.exchange.*;
+                        </Export-Package>
+                    </instructions>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-scr-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>generate-scr-scrdescriptor</id>
+                        <goals>
+                            <goal>scr</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+</project>
```

Comment:
add new line

Reply:
Fixed in 8693d5454cb05015e9d9614c56a1d09bdf8a8997